src/components/Search/SearchPageHeader.tsx

```diff
@@ -22,11 +22,12 @@ type SearchHeaderProps = {
     selectedItems?: SelectedTransactions;
     clearSelectedItems?: () => void;
     hash: number;
+    setDeleteModalVisible: (isVisible: boolean) => void;
 };
 
 type SearchHeaderOptionValue = DeepValueOf<typeof CONST.SEARCH.BULK_ACTION_TYPES> | undefined;
 
-function SearchPageHeader({query, selectedItems = {}, hash, clearSelectedItems}: SearchHeaderProps) {
+function SearchPageHeader({query, selectedItems = {}, hash, clearSelectedItems, setDeleteModalVisible}: SearchHeaderProps) {
```

Comment:
how about we change the style of callback name into `onDeleteAction` or `handleDeleteAction`, `onSelectDeleteAction` etc?

That way this component knows what the callback _is for_ but doesn't know _what it does_.
So right now its display modal, but in future it could be something else, like redirect to page or whatevs